docs/proposals/node-agent/agent.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package node_agent;
+
+enum WorkloadStatus {
+    WAITING = 0;
+    RUNNING = 1;
+    TERMINATED = 2;
+}
+
+message WorkloadRequest {
+    string id = 1;
+}
+
+enum WorkloadType {
+    CONTAINER = 0;
+}
+
+message WorkloadResource {
```

Comment:
```suggestion
message WorkloadResourceLimit {
```

Reply:
You want to define a generic `Resource` structure to represent both a workload resources limits and usage.